database-memory-agent/app.py

```
@@ -0,0 +1,260 @@
+    try:
+        existing_indexes = list(vector_collection.list_search_indexes("vector_index"))
+        if existing_indexes and existing_indexes[0].get("queryable"):
+            st.session_state.vector_index_ready = True
+            return True
+    except Exception as e:
+        st.error(f"Error checking vector index: {e}")
+        return False
+    
+    return False
+
+def process_pdf_upload(uploaded_file):
```

Ruff (0.14.8)

Unused function argument: `uploaded_file`

(ARG001)